8.cytopick_analysis/scripts/prediction_images_find.py

```diff
@@ -443,20 +452,22 @@ def random_cell_select(
             image_path5 = image_dir_path.joinpath(image_name5)
 
             # crop all 5 channels of the image
-            im1 = cv2.imread(image_path1.as_posix(), cv2.IMREAD_GRAYSCALE)
-            # im_crop1 = im1[min_y_box:max_y_box, min_x_box:max_x_box]
+            im1 = cv2.imread(image_path1.as_posix(), cv2.IMREAD_UNCHANGED)
+            # im1_crop = im1[min_y_box:max_y_box, min_x_box:max_x_box]
+
+            im2 = cv2.imread(image_path2.as_posix(), cv2.IMREAD_UNCHANGED)
+            # im2_crop = im2[min_y_box:max_y_box, min_x_box:max_x_box]
```

Comment:
Consider removing the commented lines if they are no longer needed.

Reply:
removed